eternalcore-core/src/main/java/com/eternalcode/core/feature/afk/AftPlaceholderSetup.java

```diff
@@ -1,18 +1,39 @@
 package com.eternalcode.core.feature.afk;
 
+import com.eternalcode.core.injector.annotations.Inject;
 import com.eternalcode.core.injector.annotations.component.Controller;
 import com.eternalcode.core.publish.Subscriber;
 import com.eternalcode.core.publish.event.EternalInitializeEvent;
 import com.eternalcode.core.placeholder.PlaceholderRegistry;
 import com.eternalcode.core.placeholder.PlaceholderReplacer;
 import com.eternalcode.core.publish.Subscribe;
+import com.eternalcode.core.translation.Translation;
+import com.eternalcode.core.translation.TranslationManager;
+import com.eternalcode.core.viewer.ViewerService;
 
 @Controller
 class AftPlaceholderSetup implements Subscriber {
 
+    private final TranslationManager translationManager;
+    private final ViewerService viewerService;
+
+    @Inject
+    AftPlaceholderSetup(TranslationManager translationManager, ViewerService viewerService) {
+        this.translationManager = translationManager;
+        this.viewerService = viewerService;
+    }
+
     @Subscribe(EternalInitializeEvent.class)
     void setUpPlaceholders(PlaceholderRegistry placeholderRegistry, AfkService afkService) {
-        placeholderRegistry.registerPlaceholder(PlaceholderReplacer.of("afk", player -> String.valueOf(afkService.isAfk(player.getUniqueId()))));
+        placeholderRegistry.registerPlaceholder(PlaceholderReplacer.of(
+            "afk",
+            player -> String.valueOf(afkService.isAfk(player.getUniqueId()))));
+        placeholderRegistry.registerPlaceholder(PlaceholderReplacer.of(
+            "afk_text",
```

Comment:
suffix `_text` doesn't describe what the placeholder does

Reply:
give me a better idea, please

Reply:
maybe `afk_status` ?

Reply:
then the placeholder name of another placeholder %eternalcore_afk% doesn't make sense, one is supposed to return formatted status and the other returns a boolean